### PR TITLE
feat: fakeapi and data models & bugfix: screen freeze on signout

### PR DESCRIPTION
**Pull request by nayakastha**
This Pull Request introduces the fake API and the data models that are essential for the development of the UI and ease in wireframing.

also closes issue #26 

### Build Status
[pr-build-test](https://github.com/dscnitrourkela/project-maple/actions/workflows/pull_request.yml)

### CODE REVIEW

**Comment by nayakastha** [comment]
Screenshot (181)
The page transaction on signing out happens without error.

**Review by abhibhaw — CHANGES_REQUESTED** [review]
WhatsApp Image 2021-06-27 at 6 05 53 PM
There is some error with logout flow

**Comment by nayakastha** [comment]
> image image
> Hello @nayakastha , That's a great improvement to the code. Have a pinch of paitence while the reviewer gets impressed by the changes you made. Here are some doggos for company while you are waiting for the merge and marching ahead with your Hacktoberfest Contributions, Check your [Dashboard](https://hacktoberfest.digitalocean.com/profile) for more information on Hacktoberfest. Stay safe 🚀 .

Woohoo!! @woofy is fixed

File: aptiche/lib/main.dart [hunk 1]
```diff
@@ -29,10 +30,17 @@ class MyApp extends StatelessWidget {
               title: 'APTI-CHE',
               debugShowCheckedModeBanner: false,
               theme: appTheme(),
+              routes: <String, WidgetBuilder>{
```
Comment (abhibhaw) [inline aptiche/lib/main.dart:33]: @nayakastha consider moving routes to a separate file

**Review by abhibhaw — CHANGES_REQUESTED** [review]
Stucking on Splash Screen

**Review by abhibhaw — APPROVED** [review]
🎉🎉

**Pull request merged**